New version: Catlab v0.16.20

JuliaRegistrator opened the pull request:
- Registering package: Catlab
- Repository: https://github.com/AlgebraicJulia/Catlab.jl
- Created by: @quffaro
- Version: v0.16.20
- Commit: d36b61734fc5c816e54b675b6e4a714b3be5c794
- Git reference: HEAD
- Description: A framework for applied category theory in the Julia language

quffaro commented:
[noblock] The registry consistency for v1.6 is failing due to a server issue. I assume this job would pass if it were re-run, but I do not have permissions to do so. Can this job be re-run?
![image](https://github.com/user-attachments/assets/a23c834e-b345-4467-a4a9-30edf7ab0e70)

JuliaHubRegistrator committed:
New version: Catlab v0.16.20

UUID: 134e5e36-593f-5add-ad60-77f754baafbe
Repo: https://github.com/AlgebraicJulia/Catlab.jl.git
Tree: 2d72539fc18951d28d13ab3f3152bad6df0ca348

Registrator tree SHA: d5716b7a540e5fbc43640c2fff2906fe50e9525a

quffaro commented:
Testing whether this comment will retrigger

goerz commented:
[noblock] No, sorry, I don’t have the necessary permissions for that. The `#pkg-registration` on Slack would probably be the best place to find someone

giordano commented:
As usual, you can trigger a new build yourself by registering the same version again (same version number, same commit, just exactly repeat whatever you did before)

[noblock]
